Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -9,10 +9,10 @@
 from binascii import hexlify
 from copy import deepcopy
 from shutil import rmtree
+from twisted.internet import reactor
```

Comment:
this should go one line down

Also, any decent code editor could fix that for you without even thinking :)